<?xml version="1.0" encoding="UTF-8"?>
<rss version="2.0">
	<channel>
		<title><![CDATA[Latest posts for the topic "zenoss core 3.1.0 trên centos 5.9"]]></title>
		<link>/hvaonline/posts/list/24.html</link>
		<description><![CDATA[Latest messages posted in the topic "zenoss core 3.1.0 trên centos 5.9"]]></description>
		<generator>JForum - http://www.jforum.net</generator>
			<item>
				<title>zenoss core 3.1.0 trên centos 5.9</title>
				<description><![CDATA[ Chào các bác !
Em triển khai zenoss core trên centos 5.9
Version:
<blockquote>Zenoss Zenoss 3.1.0
OS Linux (x86_64) 2.6.18 (Linux gnc.com 2.6.18-371.1.2.el5 #1 SMP Tue Oct 22 12:51:53 EDT 2013 x86_64)
Zope Zope 2.12.1
Python Python 2.6.2
Database MySQL 5.0.95 (Ver 5.0.95)
RRD RRDtool 1.3.9
Twisted Twisted 8.1.0
PyNetSnmp PyNetSnmp 0.28.14
WMI Wmi 1.3.13&nbsp;
		</blockquote>
Khi em add winsever vào để giám sát :
<blockquote>Traceback (most recent call last):
File "/opt/zenoss/Products/DataCollector/zendisc.py", line 29, in
from Products.DataCollector.zenmodeler import ZenModeler
File "/opt/zenoss/Products/DataCollector/zenmodeler.py", line 38, in
from SnmpClient import SnmpClient
File "/opt/zenoss/Products/DataCollector/SnmpClient.py", line 22, in
from pynetsnmp.twistedsnmp import snmpprotocol
File "/opt/zenoss/lib/python/pynetsnmp/twistedsnmp.py", line 1, in
import netsnmp
File "/opt/zenoss/lib/python/pynetsnmp/netsnmp.py", line 43, in
lib = CDLL(find_library('netsnmp'), RTLD_GLOBAL)
File "/opt/zenoss/lib/python/ctypes/__init__.py", line 312, in __init__
self._handle = _dlopen(self._name, mode)
OSError: libnetsnmp.so.10: cannot open shared object file: No such file or directory
Job completed at 2013-12-03 20:08:22. Result: failure.&nbsp;
		</blockquote>
Vào kiểm tra daemons thì thấy zentrap, zenmodeler, zenperfsnmp stop.
Em start lên không được.
View log:
<blockquote>Error reading zenmodeler log file '/opt/zenoss/log/zenmodeler.log':
[Errno 2] No such file or directory: '/opt/zenoss/log/zenmodeler.log'&nbsp;
		</blockquote>
Các bác cho em hỏi tại sao lại xảy ra lỗi này và cách khắc phục.
Em xin cảm ơn ạ !]]></description>
				<guid isPermaLink="true">/hvaonline/posts/list/45334.html#279146</guid>
				<link>/hvaonline/posts/list/45334.html#279146</link>
				<pubDate><![CDATA[Tue, 3 Dec 2013 23:55:22]]> GMT</pubDate>
				<author><![CDATA[ CrashOverride1102]]></author>
			</item>
			<item>
				<title>zenoss core 3.1.0 trên centos 5.9</title>
				<description><![CDATA[ Sao bạn không dùng Nagios nhỉ????]]></description>
				<guid isPermaLink="true">/hvaonline/posts/list/45334.html#279283</guid>
				<link>/hvaonline/posts/list/45334.html#279283</link>
				<pubDate><![CDATA[Sun, 22 Dec 2013 21:51:06]]> GMT</pubDate>
				<author><![CDATA[ hotdog019]]></author>
			</item>
	</channel>
</rss>
